featuretools/computational_backends/utils.py

```diff
@@ -55,13 +55,21 @@ def datetime_round(dt, freq):
     if not freq.is_absolute():
         raise ValueError("Unit is relative")
 
-    if freq.unit == 'm':
-        unit = 't'
+    # TODO: multitemporal units
```

Comment:
Add exception for multi-temporal case

Reply:
I think the assertion in line 72 should cover this case. Does it need to be changed to an exception?